src/models/GitConfig.js

```diff
@@ -151,8 +166,9 @@ export class GitConfig {
   }
 
   async get(path, getall = false) {
+    const normalizedPath = normalizePath(path).path
     const allValues = this.parsedConfig
-      .filter(config => config.path === path.toLowerCase())
+      .filter(config => config.path === normalizedPath)
```

Comment:
The `path` property in the `config` object is also the normalized path where the `section` and `name` are lowercased, but not the `subsection`.

So if we have this config file

```
[Foo "BAR"]
    BAZ = foobarbaz
```

The `config.path` will be `foo.BAR.baz`. Whatever `path` we provide to this method, we want to normalize that input to lowercase the section and name. So if we provide `FOO.BAR.BAZ`, the normalized path will be `foo.BAR.baz` matching the value in the config file.